File: Selenium/Activities/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form/")
    print(f"The title of the main page is: {driver.title}")
    username = driver.find_element(By.XPATH,"//*[@id='username']")
    username.send_keys("admin")
    password = driver.find_element(By.XPATH,"//*[@id='password']")
    password.send_keys("password")
    login = driver.find_element(By.XPATH,"//button[text()='Submit']")
    login.click()

    message = driver.find_element(By.XPATH,"//h1[contains(text(),'Success')]")
    Message = message.text
    print(f"The title of the new page is: {Message}")

    driver.quit()

File: TestNG/Activities/Activity3.java
package testng;

import static org.testng.Assert.assertEquals;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity3 {
	WebDriver driver;
	
	@BeforeClass
	public void setUp() {
		driver = new FirefoxDriver();
		driver.get("https://training-support.net/webelements/login-form/");
	}
	
	@Test
	public void credentials() {
		WebElement username = driver.findElement(By.xpath("//*[@id='username']"));
		username.sendKeys("admin");
		
		WebElement password = driver.findElement(By.xpath("//*[@id='password']"));
		password.sendKeys("password");
		
		WebElement login = driver.findElement(By.xpath("//button[text()='Submit']"));
		login.click();
		
		WebElement message = driver.findElement(By.xpath("//h2[contains(text(),'Welcome Back, Admin!')]"));
		String Message = message.getText();
		
		assertEquals("Welcome Back, Admin!",Message,"The confirmation message is wrong");
		
	}
	
	
	@AfterClass
	public void tearDown() {
		driver.quit();
	}

}
